OpenOversight/app/config.py

```diff
@@ -71,7 +71,9 @@ class ProductionConfig(BaseConfig):
 
     @classmethod
     def init_app(cls, app):  # pragma: no cover
-        config.init_app(app)
+        # ????????
+        # config.init_app(app)
+        pass
```

Comment:
Is this the bit of code that was broken you mentioned?

Reply:
Yes, in this scope `config` is a dictionary and so I cannot tell what this was meant to do